legacy/.env

```diff
@@ -7,3 +7,4 @@ JENKINS_AGENT_VERSION=3206.vb_15dcf73f6a_9
 JAVA_JDK_VERSION=11
 
 GCC_LEGACY_VERSIONS=5,7,9
+#CLANG_LEGACY_VERSIONS=
```

Comment:
You could ignore clang, we have no intention to bring it back in legacy. And, Legacy should be dead with Conan 1.x